source/Core/Utils.php

```diff
@@ -1069,17 +1069,28 @@ protected function _simpleRedirect($sUrl, $sHeaderCode)
     }
 
     /**
-     * Redirects to shop offline page
-     *
+     * Shows offline page
+     * @depricated please use showOfflinePage
```

Comment:
We have a typo here. It should be "deprecated", no "depricated".
We also need the version since when piece of code is deprecated, so the deprecation message would be something like this:
`@deprecated since v.5.3.0 (2016-06-28); Use Utils::showOfflinePage().`